File: snippets/node/view_subaccount_activity.js
const mailchimp = require("mailchimp-transactional")("YOUR_API_KEY");

async function run() {
  const response = await mailchimp.subaccounts.info({
    id: "UNIQUE_ACCOUNT_ID"
  });
  console.log(
    `This subaccount has sent ${response.sent_total} emails. Its status is ${
      response.status
    }.`
  );
}
run();

File: snippets/python/view_subaccount_activity.py
import mailchimp_transactional as MailchimpTransactional
from mailchimp_transactional.api_client import ApiClientError

mailchimp = MailchimpTransactional.Client('YOUR_API_KEY')

def run():
  try:
    response = mailchimp.subaccounts.info({
      "id": "UNIQUE_ACCOUNT_ID"
    })
    print("This subaccount has sent: {} emails. Its status is: {}".format(response['sent_total'], response['status']))
  except ApiClientError as error:
    print('An exception occurred: {}'.format(error.text))

run()